src/solver/writer/i_writer.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <yuni/core/string.h>
+#include <antares/inifile.h>
+
+#include <memory>
+#include <string>
+
+namespace Antares
+{
+namespace Solver
+{
+class IResultWriter
+{
+public:
+    using Ptr = std::shared_ptr<IResultWriter>;
+    virtual void addJob(const std::string& entryPath, Yuni::Clob& entryContent) = 0;
```

Comment:
Maybe you checked it, but I does not seem that we need **addJob** in **Yuni::Clob** version, or we can avoid it. I did not make a full check...
In case I'm right, we would get rid of a method and get rid of the **Yuni** dependance

Reply:
Sadly, it's needed.

Reply:
> Functions **addJob** actually flushes the results on disk, into a file format or another (.txt or .zip). Why calling them **addJob** rather than **flush** or **save** or **dump** ?

Because the file content isn't saved immediately, or flushed.

It's only written immediately if the "files" version is used.

Reply:
ok